catalog/cloudwatch-logs-policies.yaml

```
@@ -1,8 +1,20 @@
-- sid: "DenyDeletingCloudWatchLogs"
+- sid: "DenyCloudWatchDeletingLogs"
   effect: "Deny"
   actions:
     - "ec2:DeleteFlowLogs"
     - "logs:DeleteLogGroup"
     - "logs:DeleteLogStream"
   resources:
     - "*"
+
+- sid: "DenyDisablingCloudwatch"
```

```suggestion
- sid: "DenyDisabling CloudWatch"
```

let's use the canonical name